cli/cmd/alpha_stat.go

```diff
@@ -0,0 +1,269 @@
+				if toResource != nil {
+					metrics, err := smimetrics.GetTrafficMetricsEdgesList(k8sAPI, target.GetNamespace(), kind, name, nil)
+					if err != nil {
+						return err
+					}
+					renderTrafficMetricsEdgesList(metrics, w, toResource)
+				} else {
+					metrics, err := smimetrics.GetTrafficMetrics(k8sAPI, target.GetNamespace(), kind, name, nil)
+					if err != nil {
+						return err
+					}
+					renderTrafficMetrics(metrics, w)
+				}
+			} else {
+				if toResource != nil {
+					return errors.New("the --to flag requires that the target resource name be specified")
+				}
+				metrics, err := smimetrics.GetTrafficMetricsList(k8sAPI, target.GetNamespace(), kind, nil)
+				if err != nil {
+					return err
+				}
+				renderTrafficMetricsList(metrics, w)
+			}
+
+			w.Flush()
+			fmt.Print(buffer.String())
+			return nil
+		},
+	}
+
+	statCmd.PersistentFlags().StringVarP(&options.namespace, "namespace", "n", options.namespace, "Namespace of the specified resource")
+	statCmd.PersistentFlags().StringVar(&options.toResource, "to", options.toResource, "If present, restricts outbound stats to the specified resource name")
+
+	return statCmd
+}
+
+func buildToResource(namespace, to string) *public.Resource {
+	toResource, err := util.BuildResource(namespace, to)
+	if err != nil {
+		log.Debugf("Invalid to resource: %s", err)
+		return nil
+
+	}
+	log.Debugf("Using to resource: %v", toResource)
+	return &toResource
+}
+
+func renderTrafficMetrics(metrics *smimetrics.TrafficMetrics, w *tabwriter.Writer) {
+	renderTrafficHeaders(w, false)
+	for _, col := range metricsToRow(metrics, false) {
+		fmt.Fprint(w, col)
+		fmt.Fprint(w, "\t")
+	}
+	fmt.Fprint(w, "\n")
+}
+
+func renderTrafficMetricsList(metrics *smimetrics.TrafficMetricsList, w *tabwriter.Writer) {
+	renderTrafficHeaders(w, false)
+	for _, row := range metrics.Items {
+		row := row // Copy to satisfy golint.
+		for _, col := range metricsToRow(&row, false) {
+			fmt.Fprint(w, col)
+			fmt.Fprint(w, "\t")
+		}
+		fmt.Fprint(w, "\n")
+	}
+}
+
+func renderTrafficMetricsEdgesList(metrics *smimetrics.TrafficMetricsList, w *tabwriter.Writer, toResource *public.Resource) {
+	outbound := toResource != nil
+	renderTrafficHeaders(w, outbound)
+	for _, row := range metrics.Items {
+		row := row // Copy to satisfy golint.
+		if row.Edge.Direction != "to" {
+			continue
+		}
+		if toResource != nil && toResource.GetName() != "" &&
+			(row.Edge.Resource.Name != toResource.GetName() || row.Edge.Resource.Namespace != toResource.GetNamespace()) {
+			log.Debugf("Skipping edge %v", row.Edge.Resource)
+			continue
+		}
+		for _, col := range metricsToRow(&row, outbound) {
+			fmt.Fprint(w, col)
+			fmt.Fprint(w, "\t")
+		}
+		fmt.Fprint(w, "\n")
+	}
+}
+
+func renderTrafficHeaders(w *tabwriter.Writer, outbound bool) {
+	headers := []string{}
+	if outbound {
+		headers = append(headers, "FROM", "TO")
+	} else {
+		headers = append(headers, "NAME")
+	}
+	headers = append(headers,
+		"NAME",
```

Comment:
If `outbound` is `false`, the `NAME` header is being printed twice:
```bash
$ bin/go-run cli alpha stat -n emojivoto deploy                                                                                                                                                                                               
                                                           
       NAME      NAME   SUCCESS    RPS   LATENCY_P50   LATENCY_P90   LATENCY_P99                                                                                                                                                              
     voting    84.21%    1.0rps    1ms           2ms           2ms                                                     
        web    93.68%    4.7rps    3ms          10ms          19ms                                                     
      emoji   100.00%    1.9rps    1ms           1ms           1ms                                                     
   vote-bot    94.74%    1.9rps   10ms          18ms          20ms     
```

Reply:
good catch, thanks!